fairseq/models/text_to_speech/hub_interface.py

```diff
@@ -137,3 +137,49 @@ def predict(
     ) -> Tuple[torch.Tensor, int]:
         sample = self.get_model_input(self.task, text, speaker, verbose=verbose)
         return self.get_prediction(self.task, self.model, self.generator, sample)
+
+
+class VocoderHubInterface(nn.Module):
```

Comment:
Can you add a comment (or rename) mentioning that this is currently for unit vocoder only (there are differences in spectrogram vocoder, for example, we normally use 22khz sample rate)?